WordPress/Classes/Models/Blog.h

```diff
@@ -64,6 +64,14 @@ typedef NS_ENUM(NSUInteger, BlogFeature) {
 @property (nonatomic, assign, readwrite) BOOL isHostedAtWPcom;
 @property (nonatomic, strong, readwrite) NSString *icon;
 /**
+ Flags whether the current user is an admin on the blog.
+ 
+ @warn `isAdmin` is only being set for wpcom sites and checked when bumping page
+ views in the reader detail.
+ */
+@property (nonatomic, assign, readwrite) BOOL isAdmin;
```

Comment:
assign and readwrite are defaults but since we're already doing it elsewhere in the .h might as well keep it :wink: